packages/react-positioning/src/usePopper.ts

```diff
@@ -124,48 +125,65 @@ export function usePopper(
       arrowRef.current,
     );
 
+    // Container is always initialized with `position: fixed` to avoid scroll jumps
+    // Before computing the positioned coordinates, revert the container to the deisred positioning strategy
     Object.assign(containerRef.current.style, { position: strategy });
     computePosition(target, containerRef.current, { placement, middleware, strategy }).then(
       ({ x, y, middlewareData, placement: computedPlacement }) => {
-        if (!containerRef.current) {
-          return;
-        }
-
-        containerRef.current.setAttribute('data-popper-placement', computedPlacement);
-        Object.assign(containerRef.current.style, {
-          // Layer acceleration can disable subpixel rendering which causes slightly
-          // blurry text on low PPI displays, so we want to use 2D transforms
-          // instead
-          transform:
-            (targetDocument?.defaultView?.devicePixelRatio || 1) <= 1
-              ? `translate(${x}px, ${y}px)`
-              : `translate3d(${x}px, ${y}px, 0)`,
-          position: strategy,
-        });
-
-        containerRef.current.removeAttribute(DATA_POSITIONING_INTERSECTING);
-        if (middlewareData.intersectionObserver.intersecting) {
-          containerRef.current.setAttribute(DATA_POSITIONING_INTERSECTING, '');
-        }
+        const applyDataAttributes = () => {
+          if (!containerRef.current) {
+            return;
+          }
+          containerRef.current.setAttribute('data-popper-placement', computedPlacement);
```

Comment:
Should add a constant for this? Like `DATA_POSITIONING_INTERSECTING`

Reply:
yeah this was an oversight